web/blog/2024-10-23-pilcrow-os-maintainers-interview.md

```diff
@@ -0,0 +1,92 @@
+
+Maintaining various projects definitely helps with avoiding burnout. My projects aren’t that big in scale and actual bug reports are rare, so I don’t worry about switching my focus to a different project or taking a break for a week or two.
+
+I also see open source as just a hobby. I'll still try to address bugs quickly, but I don't rush myself to add new features.
+
+![relax gif](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExcWF3ZGRwNjFmYmVzanM0d3ZydzNjd2w5bTlxMWlqd294aWwzNmZzMiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/q10hztkXiFFmhBbH2l/giphy.gif)
+- **When collaborating on open-source, how do you manage community contributions and keep other developers involved in a meaningful way?**
+
+I don't think my projects are particularly good examples of open source collaborations. I usually ask people to open GitHub issues for new features and I'm not really open to pull requests except for bug and typo fixes. I like to take my time experimenting with various API designs and I personally find it easier if I do the initial design phase by myself. There's also a certain flavor of code I like and I really want to keep my code "beautiful" (yes, I know it sounds dumb).
+
+To be honest, I don't like pull requests in general since I feel like I wasted the contributor's time if I reject it, even if the code is objectively awful.
+
+I'm well aware that this closed approach doesn't work in bigger projects, but working on open source is ultimately just a hobby, so I think I should be able to have some fun with it.
+
+Most of the community aspect comes from Discord, and I really appreciate people who are active daily and answer questions there. Honestly, it's probably some of the most valuable contributions I receive.
+
+- **What would you do differently in the development and management of your projects, knowing what you know now about open-source and developer tools?**
+
+Defining the scope and goals of a project early on. I don’t blame myself for not thinking much ahead when starting Lucia since it was literally my first open source library, but it would’ve made development on subsequent major releases much easier. There’s only so many features you can add to a project before the code becomes a mess. You can’t appease everyone, so pick a lane and be the best for that purpose.
+
+I also think that libraries that are flexible in how and where they can be used, instead of flexible in their behavior, are more predictable and easier to work with. Clearly defining the line between library and user responsibility is going to make it much more enjoyable to use.
+
+- **What is your favorite type of ramen?**
+
+Probably tonkotsu, specifically from Kyushu (the third-largest island of Japan's four main islands). The broth is based on pork bones, which gives it a creamy and rich texture, and the noodle has a bite to it (like al dente pasta).
+
+![Tonkatsu ramen](https://prd-static.gltjp.com/glt/data/article/21000/20290/20230407_103858_503a919a_w1920.webp)
```

Comment:
Maybe I'd add some sort of CTA in the end here. People who read all the way through will probably be easily engaged. Sth like "Thanks for reading! This is our series of posts on open source creators. You can read more stories here and if you'd like to stay in touch subscribe here/follow us on X/..."